### ISSUE TITLE
[Bug]: 'Hide Thanks button' doesn't keep 'Thanks' hidden

### ISSUE THREAD

**Issue by sliceanddice9** [sapondanaisriwan/AdashimaaTube#102]
### Describe the bug

When activating the 'Hide Thanks button' - the **_'THANKS'_** button only stays hidden until you refresh the page. After you immediately refresh the page the **_'THANKS'_** button re-appears. This also applies to any other videos you watch as well - the  **_'THANKS'_** button is not  hidden.   

### What browser are you using?

Mozilla Firefox. Version: 112.0.2 (64 Bit)

### What version of AdashimaaTube are you using?

v2.2.6

**Comment by sliceanddice9** [comment]
Updated Firefox to 113.0.1 and the **'Thanks'** button not staying hidden remains.

**Comment by sapondanaisriwan** [comment]
I think the "thanks" button on your account has a different SVG path. Could you right-click the thanks button, select inspect and copy and paste in this issue?
![chrome_17OTbzESu1](https://github.com/sapondanaisriwan/AdashimaaTube/assets/64634605/1b57c2e8-5434-4ffa-83e0-d4670c471958)

**Comment by sliceanddice9** [comment]
After right-clicking the "thanks" button and then selecting "inspect" via the 'Search HTML' field I typed: **path.style-scope.yt-icon** and this is what was highlighted in blue:


> <path d="M12.7,12l6.6,6.6l-0.7,0.7L12,12.7l-6.6,6.6l-0.7-0.7l6.6-6.6L4.6,5.4l0.7-0.7l6.6,6.6l6.6-6.6l0.7,0.7L12.7,12z" class="style-scope yt-icon"></path>

**Comment by sapondanaisriwan** [comment]
let me know if this [update](https://github.com/sapondanaisriwan/sapondanaisriwan/AdashimaaTube/raw/master/AdashimaaTube.user.css) works for you.

**Comment by sapondanaisriwan** [comment]
https://github.com/sapondanaisriwan/AdashimaaTube/raw/master/AdashimaaTube.user.css

**Comment by sliceanddice9** [comment]
> Please let me know if this [update](https://github.com/sapondanaisriwan/AdashimaaTube/raw/master/AdashimaaTube.user.css) works for you.

I tried reinstalling the style with that updated version of AdashimaaTube by overwriting but sadly it's still not working with the permanent removal of the **_'THANKS'_** button.

**Comment by sliceanddice9** [comment]
[sapondanaisriwan](https://github.com/sapondanaisriwan)  upon further investigation I've worked out that what is preventing the **_'THANKS'_** button from actually staying hidden has to do with the uBlock Origin web browser ad blocking extension. When enabling the AdGuard add-on for the YouTube website the **_'THANKS'_** button is indeed removed from sight (as it should be). 

Would you happen to know what I need to configure within the uBlock Origin add-on itself to get it to cooperate with the removal of the **_'THANKS'_** button for your **AdashimaaTube** styles?

**Comment by sliceanddice9** [comment]
Closing this because I found a uBlock Origin filter that successfully keeps the **_'THANKS'_** button element removed from the UI. Thank you for your assistance [sapondanaisriwan](https://github.com/sapondanaisriwan) and keep up the amazing work you are doing with  **AdashimaaTube**. YouTube has never looked or functioned this good with how highly customizable the style is. I've blended elements from the old UI and new UI and couldn't be happier.
